shell/platform/fuchsia/flutter/platform_view_unittest.cc

```diff
@@ -143,13 +147,25 @@ class MockPlatformViewDelegate : public flutter::PlatformView::Delegate {
   const flutter::ViewportMetrics& metrics() const { return metrics_; }
   int32_t semantics_features() const { return semantics_features_; }
   bool semantics_enabled() const { return semantics_enabled_; }
+  const std::vector<std::unique_ptr<flutter::PointerDataPacket>>&
+  pointer_packets() const {
+    return pointer_packets_;
+  }
+  // Unlike the other getters, this one hands data ownership to the caller.
+  std::vector<std::unique_ptr<flutter::PointerDataPacket>>
+  obtain_pointer_packets() {
```

Comment:
The convention we've been using here is TakePointerDataPackets.  Take, and camel-case (to indicate a sife-effect)